In [2]:
#Pandas Assignment

The assignment contains data of the obesity of patients in a hospital in the file obes-phys-acti- diet.xls. The first file consists of obesity count of males and females of patients during different years. The second file consists of age breakage of obesity count.

1. Write a Python program to read .xls file through Excelfiles.

In [43]:
import pandas as pd
import chardet
import os
#print(os.getcwd())
#with open(r'obes-phys-acti-diet.xls', 'rb') as f:
 #   result = chardet.detect(f.read())
  #  print(result)    
data1 = pd.ExcelFile('obes-phys-acti-diet.xls')


Ques 2.Print sheet names

In [45]:
print(data1.sheet_names)

['Chapter 7', '7.1', '7.2', '7.3', '7.4', '7.5', '7.6', '7.7', '7.8', '7.9', '7.10']


ques 3.
. Write a Python program to create a list of six columns:
 Year
 Total
 Males
 Females  Nan
 None

In [50]:
col = pd.DataFrame([['Year'],['Total'],['Males'],['Females'],['Nan'],['None']])
print(col)

         0
0     Year
1    Total
2    Males
3  Females
4      Nan
5     None


Ques 4. Write a python program to parse the data and read the six columns present in 7.1
sheet of data and skip first six rows and set skipfooter=14 and the column names as
the list defined above. Assign the dataframe to data_gender.